input/templates/partials/common/footer.hbs

```diff
@@ -4,56 +4,61 @@
       <div class="col-sm-2">
         <div class="footer-col">
           <ul>
-            <li class="footer-title hidden-xs">{{footer.customerCare.text}}</li>
-            {{#each footer.customerCare.list}}
-              <li><a href="{{url}}">{{text}}</a></li>
-            {{/each}}
+            <li class="footer-title hidden-xs">{{i18n "footer.customerCare"}}</li>
+            <li><a href="{{@root.meta._links.contactUs.href}}">{{i18n "footer.contactUs"}}</a></li>
```

Comment:
Does it make sense to wrap it in an`if` so if `root.meta._links.contactUs.href` is not present, the link would not be shown?